### PR TITLE
(maint) mirror aix packages used in tests

### PR DESCRIPTION
**Pull request by ciprianbadescu**

### CODE REVIEW

**Comment by ciprianbadescu** [comment]
jenkins please test this on aix-7.1-ppc

**Comment by ciprianbadescu** [comment]
jenkins please test this on aix71-POWERfa

**Comment by ciprianbadescu** [comment]
jenkins please test this on aix71-POWERfa,aix72-POWERfa

File: acceptance/tests/validate_vendored_ruby.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ def package_installer(agent)
   # need to manually set up the package installer for these platforms.
   case agent['platform']
   when /aix/
-    lambda { |package| on(agent, "rpm -Uvh #{package}") }
+    lambda { |package| on(agent, "curl -O #{package} && rpm -Uvh `basename #{package}`") }
```
Comment (joshcooper) [inline acceptance/tests/validate_vendored_ruby.rb:13]: Should we try to delete an existing package before curling, for example when re-running the test?

**Comment by ciprianbadescu** [comment]
Passing ad-hoc: 
image

**Review by gimmyxd — APPROVED** [review]

**Pull request merged**